### PR TITLE
Prepare migrations for 2.1b1.

### PR DESCRIPTION
**Pull request by normanjaeckel**
Agenda, Assignments, Core, Mediafiles, Users.

Topics are already done.

Motions are still missing. Here I want to wait for some rework of motion change recommendation.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Please ping me for final review.

**Comment by normanjaeckel** [comment]
Ping. I can refactor motions later.

**Comment by emanuelschuetze** [comment]
Tested:
* remove database
* checkout current master and run "python manage.py start" (current migrations run successfully)
* checkout your branch and run "python manage.py start"
-> Migrate process ends with error:
```
django.db.utils.OperationalError: table "core_projectiondefault" already exists
```

**Comment by normanjaeckel** [comment]
Of course the migration process only works from 2.0 to my branch, not from any dev commit in between.

**Comment by normanjaeckel** [comment]
@emanuelschuetze All done and ready to merge. Please also test migration path from 2.0 to this version. After this is merged everybody has to renew databases from commits in-between.

**Comment by emanuelschuetze** [comment]
@normanjaeckel I added a new commit to remove old countdowns and messages (created with 2.0) to prevent JS errors ("unknown slide" etc.).
Please check my commit carefully. I tested it successfully with the 2.0 demo database.

File: openslides/core/migrations/0002_misc_features.py [hunk 1]
```diff
@@ -46,6 +46,21 @@ def name_default_projector(apps, schema_editor):
...
+def remove_old_countdowns_messages(apps, schema_editor):
+    """
+    Remove old countdowns and messages created by 2.0 from projector elements which are unusable in 2.1.
+    """
+    Projector = apps.get_model('core', 'Projector')
+    projector = Projector.objects.get(pk=1)
+
+    projector_config = projector.config
+    for key, value in list(projector.config.items()):
+        if value.get('name') == 'core/countdown' or 'core/message':
```
Comment (normanjaeckel) [inline openslides/core/migrations/0002_misc_features.py]: Are you sure that this condition is interpreted as `value.get('name') == ('core/countdown' or 'core/message)` and not as `(value.get('name') == 'core/countdown') or 'core/message`?

I propose to use: `value.get('name') in ('core/countdown', 'core/message')`.

Comment (emanuelschuetze) [inline openslides/core/migrations/0002_misc_features.py]: Done (I used your proposed line).

**Pull request merged**